helm/url-shortener/custom-values.yaml

```diff
@@ -1,4 +1,6 @@
 kutt:
+  namespace: url-shortener
```

Comment:
you are placing this namespace inside the kutt subchart values.... I would place it at your chart's level